zppy/templates/e3sm_diags.bash

```diff
@@ -67,64 +67,6 @@ create_links_climo_diurnal()
   cd ..
 }
 
-create_links_ts()
```

Comment:
1) We completely remove the `create_links` functions. No more using `cdscan` to compile a list of files into an xml.